src/app/components/infinite-scroll/index.html

```diff
@@ -0,0 +1,29 @@
+<sky-demo-page
+  pageTitle="Infinite scroll">
+
+  <sky-demo-page-summary>
+    The infinite scroll component allows users to load data as the page scrolls.
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties>
+    <sky-demo-page-property
+      propertyName="hasMore"
+      defaultValue="false"
+      isOptional="true"
+    >
+    A boolean specifying whether there is more data to load. If this is false, infinite scroll will hide and stop calling the load function.
+    </sky-demo-page-property>
+
+    <sky-demo-page-property
+      propertyName="onLoad"
+      isOptional="false"
+    >
+    This is an event that fires when when the load more button is clicked or a scroll triggers a need to load more data. This will only emit an event when the hasMore property is `true` and data is not currently loading.
```

Comment:
Delete "This is an event that" and just start with "Fires" for consistency with other component properties/events. Also, delete the extra "when" and change "the load more button is clicked" to "users click the load more button" to use active voice. ... And after "triggers," I'd change "a" to "the."

Change "will only emit" to "only emits." ... Wrap "hasMore" in "stache-code" tags and change the tick marks around "true" to "stache-code" tags. ... I think we can delete "currently" without changing the meaning.